plugins/woocommerce/client/legacy/js/frontend/woocommerce.js

```diff
@@ -192,7 +192,7 @@ jQuery( function ( $ ) {
  * This function focus on the first notice message with the role="alert"
  * attribute to make sure it's announced.
  */
-function focus_populate_live_region() {
+function focus_populated_live_region() {
```

Comment:
I leveraged this PR to do a small typo fix:
`focus_populate_live_region` -> `focus_populated_live_region`